mmrazor/models/task_modules/estimators/counters/flops_params_counter.py

```diff
@@ -409,6 +411,12 @@ def add_flops_params_counter_hook_function(module):
 
             else:
                 counter_type = get_counter_type(module)
+                if counter_type not in TASK_UTILS._module_dict.keys():
+                    old_counter_type = counter_type
```

Comment:
Move the logic into `get_counter_type` and then you can also refactor `is_supported_instance`.

Reply:
Done.